### ISSUE TITLE
Version Numbering

### ISSUE THREAD

**Issue by marpierc** [OSC/Open-OnDemand#34]
Do you have an explanation of your versioning scheme?  Your documentation refers to Version 1.2 (https://osc.github.io/ood-documentation/master/infrastructure.html), but your software actually consists of many components with different versioning schemes. What constitutes 1.2?

**Comment by marpierc** [comment]
I take it your release versions are set in https://github.com/OSC/ood-apps-installer/releases.

**Comment by ericfranz** [comment]
Previously, each component and associated library followed semantic versioning as OnDemand was a collection of disparate components working together.

When we introduced a single "version" to represent a release of OnDemand and to communicate which versions of components work well together, we identified the versions of each component that make up this release in the release notes. Here is an example: https://osc.github.io/ood-documentation/master/release-notes/v1.2-release-notes.html#infrastructure

Those show both the versions and the change between the versions, as well as a link to the GitHub diff.

Soon we will release v1.3 and that will have an rpm. So yum install ondemand will install the dependent rpms as well. After this release, we will eventually move to a single repo for most components, so that installing from source is as easy as installing from rpm.
